pkg/api/plugin_resource.go

```diff
@@ -211,6 +217,11 @@ func (hs *HTTPServer) flushStream(stream callResourceClientResponseStream, w htt
 				}
 			}
 
+			// Make sure a content type always is returned in response
+			if !hasContentType && resp.Status != http.StatusNoContent {
```

Comment:
A test case for these would be nice!

Reply:
In case they don't already exist 😬 Only just saw the logic is moved from before - so can ignore otherwise

Reply:
Yep there's already a test case for that:

https://github.com/grafana/grafana/blob/41185ee9861b2cb8dcaee2b0516333d0033c6214/pkg/api/plugins_test.go#L398-L420

Reply:
Maybe a test case when status is 204 would be nice as well?